### PR TITLE
added restfulapi plugin to vdr-addon for accessing the wirbelscan plugin via jsonrpc

### PR DESCRIPTION
**Pull request by gdachs**
added restfulapi plugin to vdr-addon for accessing the wirbelscan plugin via jsonrpc.
Documentation and a less important function is still missing and will follow.

### CODE REVIEW

File: packages/multimedia/vdr-plugin-restfulapi/package.mk [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+
+PKG_NAME="vdr-plugin-restfulapi"
+PKG_VERSION="20150114180413unstable"
+PKG_REV="0yavdr0~trusty"
+PKG_ARCH="any"
+PKG_LICENSE="GPL"
+PKG_SITE="https://launchpad.net/~yavdr"
+PKG_URL="https://launchpad.net/~yavdr/+archive/ubuntu/unstable-vdr/+files/${PKG_NAME}_${PKG_VERSION}-${PKG_REV}.tar.gz"
+PKG_SOURCE_DIR="${PKG_NAME}_${PKG_VERSION}"
+PKG_DEPENDS_HOST="cxxtools:host"
```
Comment (stefansaraev) [inline packages/multimedia/vdr-plugin-restfulapi/package.mk]: this is intentional and cxxtools:host is required ?

Comment (gdachs) [inline packages/multimedia/vdr-plugin-restfulapi/package.mk]: Another copy. I suppose I don't need it, but can only check at home.

Comment (stefansaraev) [inline packages/multimedia/vdr-plugin-restfulapi/package.mk]: ok thanks, I'll do build test tonight and squash it for you if you dont mind.

Comment (gdachs) [inline packages/multimedia/vdr-plugin-restfulapi/package.mk]: sorry what means squash in this context?

Comment (stefansaraev) [inline packages/multimedia/vdr-plugin-restfulapi/package.mk]: http://git-scm.com/book/en/v2/Git-Tools-Rewriting-History

**Comment by stefansaraev** [comment]
added. thanks

**Pull request closed without merging**